packages/rs-dpp/src/state_transition/state_transitions/document/documents_batch_transition/document_transition/document_replace_transition/v0/mod.rs

```diff
@@ -352,12 +409,35 @@ impl DocumentFromReplaceTransitionV0 for Document {
 
         let id = base.id();
 
+        let requires_updated_at = document_type
```

Comment:
again similar logic to get optionally required field, that takes multiple lines

Reply:
Discussed with Ivan, this is fine.